api/server/router/container/container_routes.go

```diff
@@ -586,9 +586,15 @@ func (s *containerRouter) wsContainersAttach(ctx context.Context, w http.Respons
 		UseStdin:   true,
 		UseStdout:  true,
 		UseStderr:  true,
-		MuxStreams: false, // TODO: this should be true since it's a single stream for both stdout and stderr
+		MuxStreams: true,
 	}
 
+	// In prior API versions we were only mixing stdout/stderr streams together
+	// with no way for the client to differentiate. Setting `MuxStreams` to false
+	// keeps this old behavior.
+	if versions.LessThan(version, "1.31") {
```

Comment:
This would have to be 1.35 now; https://github.com/moby/moby/blob/master/api/swagger.yaml#L22